tools/python/categories_merge_language.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import sys
+import os
+
+if len(sys.argv) < 2:
+    print('Merges some language in categories.txt with English')
+    print('Usage: {} <lang> [path_to_categories.txt]'.format(sys.argv[0]))
+    sys.exit(1)
+
+lang = sys.argv[1]
+if len(sys.argv) > 2:
+    path = sys.argv[2]
+else:
+    path = os.path.join(os.path.dirname(sys.argv[0]), '..', '..', 'data', 'categories.txt')
+
+with open(path, 'r') as f:
+    first = True
+    langs = []
+    trans = None
+
+    def flush_langs():
+        for l in langs:
+            if trans and l[0] == 'en':
+                parts = l[1].split('|')
+                parts[0] = '{} - {}'.format(parts[0], trans)
+                l[1] = '|'.join(parts)
+            print(':'.join(l))
+
+    for line in f:
+        if len(line.strip()) == 0 or line[0] == '#':
+            if langs:
+                flush_langs()
+                langs = []
+                trans = None
+            print(line.strip())
+            first = True
+        elif first:
```

Comment:
Because of `area:highway`. Although you are right, I can just use `elif not langs`.